scripts/update-pipfile-lock

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# This script updates our Pipfile.lock using the oldest supported version of
+# Python. The version is defined in the Dockerfile.update-pipfile-lock file.
+set -euo pipefail
+
+IMAGE_NAME=ggshield-update-pipfile-lock
+
+SCRIPT_DIR=$(dirname $0)
+
+cd $SCRIPT_DIR
+docker build -t $IMAGE_NAME --build-arg UID=$UID -f Dockerfile.update-pipfile-lock .
+
+cd $SCRIPT_DIR/..
+docker run -v $PWD:/home/user/src $IMAGE_NAME
```

Comment:
nit: maybe create an entry in the Makefile to run the script easily

Reply:
Good idea :+1: